Clients/src/Swag4Net.Core/v3/SpecificationDocument.fs

```diff
@@ -0,0 +1,245 @@
+  and Server = 
+    {
+      Url:string
+      Description:string option
+      Variables:Map<string, ServerVariable> option }
+  and ServerVariable = 
+    {
+      Enum: string list
+      Default: string
+      Description: string }
+  and Path =
+    {
+      Reference: string
+      Summary:string
+      Description:string
+      Get:Operation
+      Put:Operation
+      Post:Operation
+      Delete:Operation
+      Options:Operation
+      Head:Operation
+      Patch:Operation
+      Trace:Operation
+      Servers:Server list
+      Parameters:TypeOrReference<Parameter> list
+      }
+  and Operation =
+    {
+      Tags: string list option
+      Summary: string option
+      Description: string option
+      ExternalDocs: ExternalDocumentation option
+      OperationId: string 
+      Parameters: TypeOrReference<Parameter> list option
+      RequestBody: TypeOrReference<Request> option
+      Responses: Responses
+      Callbacks: Map<string, TypeOrReference<Callback>> option
+      Deprecated: bool option
+      Security: SecurityRequirement list option
+      Servers: Server list option }
+  and Parameter =
+    {
+        Name: string
+        In: string
+        Description: string option
+        Required: bool option
+        Deprecated: bool option
+        AllowEmptyValue: bool option
+        Style: string option
+        Explode: bool option
+        AllowReserved: bool option
+        Schema: TypeOrReference<Schema> option
+        Example: Any option
+        Examples: Map<string, TypeOrReference<Example>> option
+        Content: Map<string, MediaType> option }
+  and MediaType =
+    {
+      Schema: TypeOrReference<Schema>
+      Examples: Map<string, TypeOrReference<Example>>
+      Encoding: Map<string, Encoding> }
+  and Schema =
+    {
+      Title: string option
+      AllOf: TypeOrReference<Schema> option
+      OneOf: TypeOrReference<Schema> option
+      AnyOf: TypeOrReference<Schema> option
+      Not: TypeOrReference<Schema> option
+      MultipleOf: TypeOrReference<Schema> option
+      Items: ItemType option
+      Maximum: int option
+      ExclusiveMaximum: int option
+      Minimum: int option
+      ExclusiveMinimum: int option
+      MaxLength: int option
+      MinLength: int option
+      Pattern: RegularExpression option
+      MaxItems: int option
+      MinItems: int option
+      UniqueItems: bool option
+      MaxProperties: int option
+      MinProperties: int option
+      Properties: Map<string, Schema> option
+      AdditionalProperties: AdditionalProperties option
+      Required: bool option
+      Nullable: bool option
+      Enum: Any list option
+      Format: DataTypeFormat option
+      Discriminator: Discriminator option
+      Readonly: bool option
+      WriteOnly: bool option
+      Xml: Xml option
+      ExternalDocs:ExternalDocumentation option
+      Example: Any option
+      Deprecated:bool option
+    }
+  and AdditionalProperties =
+    | B of bool
```

Comment:
Please use more explicit names

Reply:
yep i am not sure how to name union member